drivers/node/ssh/ssh.go

```diff
@@ -161,11 +158,17 @@ func (s *SSH) GetDeviceMapperCount(n node.Node, timerange time.Duration) (int, e
 
 // Init initializes SSH node driver
 func (s *SSH) Init(nodeOpts node.InitOptions) error {
+	var err error
 	s.log = nodeOpts.Logger
 	s.specDir = nodeOpts.SpecDir
 
+	execPodNamespace, err := getExecPodNamespace()
```

Comment:
Is it not possible to define pxNameSpace as global variable ? It can be part of `torpedo` structure and set it during `Init`. Why to call every time to get pxNameSpace ?

Reply:
we can't @jainsmit as torpedo can be run without Portworx as a volume driver